e2e/windows/e2e-scenario.sh

```diff
@@ -95,6 +99,11 @@ fi
 log "Upload cse packages done"
 
 log "Scenario is $SCENARIO_NAME"
+log "Windows package url is $windowsPackageURL"
+log "Windows package version is $WINDOWS_PACKAGE_VERSION"
+
+# Generate vmss cse deployment config for windows nodepool testing
+export orchestratorVersion=$WINDOWS_PACKAGE_VERSION
```

Comment:
The `orchestratorVersion` in `property-windows-template.json` is the version for the ARM template. I do have seen some transient windows vmss deployment errors using `v1.23.12` with cse package v1.25.6+. 

I think we can use the windows package version for the windows vmss deployment to reduce the chance of deployment failure. @junjiezhang1997 ?

Reply:
@junjiezhang1997 found that the $orchestratorVersion in `property-windows-template` is used to decide if it's necessary to remove the dynamic kubelet config flag that's not supported by 1.24.0+ in `baker.go`.

If 1.11 < version < 1.24, `DynamicKubeletConfig=False` in feature gates when validating windows nodebootstrapping;
if version > 1.24, this flag is simply removed.